aten/src/ATen/native/cuda/TensorModeKernel.cu

```diff
@@ -182,16 +183,19 @@ void fused_mode(
       break;
     case 128:
     case 64:
-      handle_fused_mode<128, scalar_t>(
-          grid, self, ti_values, ti_indices, slice_size, slices);
-      break;
     case 32:
     case 16:
     case 8:
     case 4:
-    case 2:
-      handle_fused_mode<32, scalar_t>(
```

Comment:
32 seems like a weird choice to me. Why have a dedicated kernel with fewer threads than there are in a single warp?

Reply:
Yeah, that's a weird choice indeed, probably they missed that `size/2`